src/integrationTest/java/uk/gov/hmcts/darts/cases/service/CaseServiceAdvancedSearchUseInterpreterTest.java

```diff
@@ -255,12 +255,12 @@ void testSearchCasesWithPermissionsGlobalTrueOnCourthouselessUserGroupUseInterpr
         // given
         var securityGroup = SecurityGroupTestData.buildGroupForRole(APPROVER);
         securityGroup.setGlobalAccess(true);
-        securityGroup.setUseInterpreter(true);
+        securityGroup.setUseInterpreter(false);
         assignSecurityGroupToUser(user, securityGroup);
 
         var securityGroupSwansea = SecurityGroupTestData.buildGroupForRoleAndCourthouse(TRANSLATION_QA, swanseaCourthouse);
         securityGroupSwansea.setGlobalAccess(false);
```

Comment:
have we reviewed all the test cases in this class even if they still pass?
Perhaps they still pass but with the new change they should fail now? i.e. we could have false positives

Reply:
We can get rid of any test case which we have written with multiple role use case.